cpp/log/cert.cc

```diff
@@ -92,6 +93,24 @@ Cert::Status Cert::LoadFromDerString(const std::string& der_string) {
   return Cert::TRUE;
 }
 
+Cert::Status Cert::LoadFromDerBio(BIO *bio_in) {
+  if (x509_ != NULL) {
+    X509_free(x509_);
```

Comment:
Hmm, not on you, but looks like we should convert this class to use a `unique_ptr` with custom deallocator for x509_, could you pop a TODO in there against me?